.github/ISSUE_TEMPLATE/working-group-proposal.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Working Group Proposal
+about: A sample template for working group proposals.
+title: WG-Proposal
+labels: pending-triage
+assignees: ''
+
+---
+
+**Description:** A short description of what the proposed WG is all about.
+
+**Impact:** The impact this WG will create.
+
+**Scope:** A draft of the initial scope that the WG will be looking into (suggestions)
+
+**WG Chairs/Leads Nominations (2/3):** Even though the WG is open to everyone we need 2-3 people to lead the efforts.
+
+**TAG Representative:** Someone from the TAG leadership team who will be coordinating all of these efforts and will help/guide wherever needed.
+
+**Diversity/Inclusivity Moderator (Optional):** Apart from the leadership team you should nominate a moderator who can review anything produced by the WG in any form and make sure it includes everyone.
+
+To Do:
+- [ ] WG Chair Nominee 1
+- [ ] WG Chair Nominee 2
+- [ ] TAG Representative
+- [ ] Moderator
```

Comment:
IMHO, it'd be nice to have a member list as well, just to acknowledge the folks behind the WG?